src/abstractdatatable/reshape.jl

```diff
@@ -202,21 +202,16 @@ function unstack(dt::AbstractDataTable, rowkey::Int, colkey::Int, value::Int)
     if T <: Nullable
         T = eltype(T)
     end
-    payload = DataTable(Any[NullableArray(T, Nrow) for i in 1:Ncol],
+    payload = DataTable(Any[NullableVector{T}(Nrow) for i in 1:Ncol],
                         map(Symbol, levels(keycol)))
-    nowarning = true
     for k in 1:nrow(dt)
         j = Int(CategoricalArrays.order(keycol.pool)[keycol.refs[k]])
         i = Int(CategoricalArrays.order(refkeycol.pool)[refkeycol.refs[k]])
         if i > 0 && j > 0
-            if nowarning && !isnull(payload[j][i])
```

Comment:
I'm not sure it needs to be obsolete. I couldn't figure out what it was testing for and also couldn't figure out how to trigger the warning it throws. We should keep it if we can figure out how to trigger it.